### ISSUE TITLE
Bitcoin Cash history

### ISSUE THREAD

**Issue by NxtChg** [bitcoincashorg/website#38]
There is this nice list of events in Bitcoin Cash:

https://www.reddit.com/r/btc/comments/7dtbew/bitcoin_cash_a_reflection_on_how_far_weve_come/

It would be a nice addition to the site as some History section somewhere.

The author is willing to provide all the proof-links if this list gets accepted, and I am willing to throw $20 to him and $30 to whoever formats it into a pull request.

What do you think? Should this be added?

**Comment by fyookball** [comment]
I like this idea.  It is factual, historical, and shows the tremendous progress.  Should be done in a nice graphical design.  

Perhaps we should also give events PRIOR to creation, like failed HK agreement, etc... even going back as far as the genesis block, and then the 2013 Bitcoin can't scale stuff from greg and peter, bitcoin xt, etc... some of the biggest milestones of the scaling debate.  if you how look how ethereum classic gives a timeline on their site, i like that too

**Comment by NxtChg** [comment]
Timeline is a good idea, only ETC's timeline is horizontal, which is hard to read.

More natural would be to just add a vertical timeline to the list, similar to github or facebook.

Maybe in reversed order so the most recent events are at the top, sort of like a simple news site of every major event in Bitcoin Cash. No articles, just events and links.

Then we can gradually add more and more history to the bottom about what led to the fork.

**Comment by fyookball** [comment]
agree on the vertical thing.  

im on the fence  about reversed order.

**Comment by fyookball** [comment]
another thing to consider is that this would make the site very long.  If it were added, should it be its own page?  At the very least it should be collapsable.

**Comment by zquestz** [comment]
I am all for this, and it should be on a separate page.

**Comment by NxtChg** [comment]
> should it be its own page?

Of course, same as the merchant list. No reason to try and cram everything into a single page.

**Comment by fyookball** [comment]
can you put this together @NxtChg ?  If you need a few bits to pay a graphic designer let me know.

**Comment by maplesyrupsucker** [comment]
Love this idea!

**Comment by NxtChg** [comment]
@fyookball The guy promised to work on it and I've never heard from him again :)

I will pester him more.

**Comment by NxtChg** [comment]
After two unsuccessful attempts, I finally found somebody to add links to the list:

https://www.reddit.com/r/btc/comments/7kk165/50_bounty_bitcoin_cash_history/

But it's a tedious job, so if anyone could throw a few bucks at him, that would be great.

**Comment by NxtChg** [comment]
The list is finished, please tip https://www.reddit.com/user/cipher_gnome some more, it was a hard job done very quickly.

**Comment by zquestz** [comment]
This is fantastic. Can't wait to see the end result. =)

**Comment by NxtChg** [comment]
@Nawsen I double-checked and cleaned up the list, so it's ready to be formatted:

https://tsbw.io/history.json

**Comment by Nawsen** [comment]
@NxtChg Cool, I will try to find some time the next days to continue working on the technical side.
